### PR TITLE
Fetch substation ID for voltage level

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java [hunk 1]
```diff
@@ -107,6 +107,11 @@ public List<ElementInfos> getVoltageLevelBuses(UUID networkUuid, String voltageL
                 .map(BusInfosMapper::toListInfos).collect(Collectors.toList());
     }
 
+    public String getVoltageLevelSubstationID(UUID networkUuid, String voltageLevelId, String variantId) {
+        Network network = getNetwork(networkUuid, PreloadingStrategy.NONE, variantId);
+        return network.getVoltageLevel(voltageLevelId).getNullableSubstation().getId();
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/network/map/NetworkMapService.java:112]:
```suggestion
          return network.getVoltageLevel(voltageLevelId).getSubstation().map(Substation::getId).orElse(null))
```

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java [hunk 2]
```diff
@@ -110,7 +110,7 @@ public List<ElementInfos> getVoltageLevelBuses(UUID networkUuid, String voltageL
 
     public String getVoltageLevelSubstationID(UUID networkUuid, String voltageLevelId, String variantId) {
         Network network = getNetwork(networkUuid, PreloadingStrategy.NONE, variantId);
-        return network.getVoltageLevel(voltageLevelId).getNullableSubstation().getId();
+        return network.getVoltageLevel(voltageLevelId).getSubstation().map(Substation::getId).orElse(null);
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/network/map/NetworkMapService.java:113]: Add a test for null

Comment (souissimai) [reply src/main/java/org/gridsuite/network/map/NetworkMapService.java:113]: done

**Review by SlimaneAmar — APPROVED** [review]
